VimeoUpload/Upload/Model/VideoSettings.swift

```diff
@@ -96,4 +112,16 @@ class VideoSettings: NSObject
         aCoder.encodeObject(self.users, forKey: "users")
         aCoder.encodeObject(self.password, forKey: "password")
     }
+    
+    // MARK : String Methods
+    
+    func trim(text: String?) -> String?
+    {
+        if let constText = text
+        {
+            return constText.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceCharacterSet())
+        }
+        
+        return nil
```

Comment:
This can just be:

`return text?.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceCharacterSet())'

After that we should be good to merge!